src/components/sbb-toast/sbb-toast.stories.tsx

```diff
@@ -0,0 +1,186 @@
+/** @jsx h */
+import events from './sbb-toast.events';
+import { Fragment, h, JSX } from 'jsx-dom';
+import readme from './readme.md';
+import { withActions } from '@storybook/addon-actions/decorator';
+import isChromatic from 'chromatic';
+import type { Meta, StoryObj, ArgTypes, Args, Decorator } from '@storybook/html';
+import type { InputType } from '@storybook/types';
+import { within } from '@storybook/testing-library';
+import { waitForComponentsReady } from '../../global/helpers/testing/wait-for-components-ready';
+
+// Story interaction executed after the story renders
+const playStory = async ({ canvasElement }): Promise<void> => {
+  const canvas = within(canvasElement);
+
+  await waitForComponentsReady(() =>
+    canvas.getByTestId('sbb-toast').shadowRoot.querySelector('.sbb-toast')
+  );
+
+  const toast = canvas.getByTestId('sbb-toast') as HTMLSbbToastElement;
+  toast.open();
+  await new Promise((resolve) => setTimeout(resolve, 2000));
+};
+
+const horizontalPosition: InputType = {
+  control: {
+    type: 'select',
+  },
+  options: ['left', 'center', 'right', 'start', 'end'],
+};
+
+const verticalPosition: InputType = {
+  control: {
+    type: 'select',
+  },
+  options: ['top', 'bottom'],
+};
+
+const dismissible: InputType = {
+  control: {
+    type: 'boolean',
+  },
+};
+
+const timeout: InputType = {
+  control: {
+    type: 'number',
+    step: 500,
+  },
+};
+
+const politeness: InputType = {
+  control: {
+    type: 'select',
+  },
+  options: ['assertive', 'polite', 'off'],
+};
+
+const iconName: InputType = {
+  control: {
+    type: 'text',
+  },
+};
+
+const disableAnimation: InputType = {
+  control: {
+    type: 'boolean',
+  },
+};
+
+const defaultArgTypes: ArgTypes = {
+  horizontalPosition,
+  verticalPosition,
+  dismissible,
+  timeout,
+  politeness,
+  iconName,
+  disableAnimation,
+};
+
+const defaultArgs: Args = {
+  horizontalPosition: 'center',
+  verticalPosition: 'bottom',
+  dismissible: false,
+  timeout: 6000,
+  politeness: 'assertive',
+  iconName: 'circle-tick-small',
+  disableAnimation: isChromatic(),
+};
+
+const toastTemplate = (args, action, contentLength = 's'): JSX.Element => (
+  <Fragment>
+    <sbb-button onClick={() => document.querySelector('sbb-toast').open()}>Show toast</sbb-button>
+    <sbb-toast
+      position={`${args.verticalPosition}-${args.horizontalPosition}` as any}
+      dismissible={args.dismissible}
+      timeout={args.timeout}
+      politeness={args.politeness}
+      icon-name={args.iconName}
+      disable-animation={args.disableAnimation}
+      data-testid="sbb-toast"
```

Comment:
would it not more stable to just ust {...args} ? for the position you could still concatenate it. But I'm also thinking if it is only one propery (position), it's better to also only provide one control for it with all the options in it. or otherwise we should make two props in the component

Reply:
I thought to have two different controls on storybook because it seemed more convenient to use them.
On the other hand, a single input is more convenient from a code perspective.

But you might be right, it might be misleading and maybe we should have a single control in storybook too. What do you think?

Reply:
I think I‘d prefer s single control.

Reply:
Got it. This way we can go back using {...args} too